OpenRA.Mods.RA2/dependencies/readme.txt

```diff
@@ -0,0 +1,5 @@
+You should copy following files out of your main OpenRA install in here:
```

Comment:
You should actually copy the assemblies from the required version into this directory as instructed.